### PR TITLE
Fix range_offset_limit debugging

### PR DESCRIPTION
**Pull request by sujiacong**

### CODE REVIEW

File: src/HttpRequest.cc [hunk 1]
```diff
@@ -615,8 +615,8 @@ HttpRequest::getRangeOffsetLimit()
     for (AclSizeLimit *l = Config.rangeOffsetLimit; l; l = l -> next) {
         /* if there is no ACL list or if the ACLs listed match use this limit value */
         if (!l->aclList || ch.fastCheck(l->aclList).allowed()) {
-            debugs(58, 4, HERE << "rangeOffsetLimit=" << rangeOffsetLimit);
             rangeOffsetLimit = l->size; // may be -1
+            debugs(58, 4, HERE << "rangeOffsetLimit=" << rangeOffsetLimit);
```
Comment (rousskov) [inline src/HttpRequest.cc]: Please remove `HERE <<` from the arguably modified debugs() line. We are trying to get rid of that macro when we have to change debugging for some other reason.

**Comment by rousskov** [comment]
I polished the future commit message (i.e., PR title and description). Do not worry about multiple commits in this PR (after you remove `HERE`) or your PR branch being a little out of date. In a few days, a [merge bot](http://lists.squid-cache.org/pipermail/squid-dev/2018-February/009366.html) should take care of that.

**Comment by yadij** [comment]
@sujiacong, can you pleas add the requested change please so we can close this ASAP.

**Comment by rousskov** [comment]
@yadij, I am happy to waive my (very minor) change request if you are in a hurry to merge this. If you want that, approve this PR "as is" and I will do the same.

**Comment by yadij** [comment]
OK to test

**Pull request closed without merging**